tests/testsuite/rust_version.rs

```diff
@@ -124,7 +124,10 @@ fn rust_version_too_high() {
         .with_status(101)
         .with_stderr(
             "error: package `foo v0.0.1 ([..])` cannot be built because it requires \
-             rustc 1.9876.0 or newer, while the currently active rustc version is [..]",
+             rustc 1.9876.0 or newer, while the currently active rustc version is [..]\n\
+             Either upgrade[..]\n\
+             cargo update[..]\n\
+             where[..]",
```

Comment:
I would prefer not to have `[..]` here, as that is obscuring the fact that the displayed syntax isn't correct.

Reply:
Fixed.